html/captive-portal/lib/captiveportal/PacketFence/Controller/Root.pm

```diff
@@ -316,6 +316,17 @@ sub getRequestLanguages : Private{
     return \@l;
 }
 
+=head2 setupStatsd
+
+setup statsd
+
+=cut
+
+sub setupStatsd : Private {
```

Comment:
Is it just me or that just do nothing at all ?

Maybe I'm missing some perl magic